.travis.yml

```diff
@@ -1,17 +1,25 @@
 sudo: required
 services:
   - docker
+addons:
```

Comment:
what does it do? can you post a link to bit of documentation which explains it? thanks

Reply:
That's a way to install PostgresSQL in a specific version. By default that `9.2`. See: https://docs.travis-ci.com/user/database-setup/#postgresql